server/server.go

```diff
@@ -50,7 +50,11 @@ func (p *PTMergeServer) Run() {
 	}
 	log.Printf("Connected to mongodb at %s\n", p.DatabaseHost)
 
-	p.Database = session.DB(p.DatabaseName)
+	// this master database session should be copied using session.Copy()
+	// before making requests to the database. This protects the connection
+	// to mongo if for any reason a database operation times out.
```

Comment:
Correct. Eventually it will get passed to the `Merger` and used during the merge process. At that time I expect the session to be copied.